samples/FXL_I2C/src/main.c

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2015 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <errno.h>
+#include <zephyr.h>
+#include <sys/printk.h>
+#include <device.h>
+#include <drivers/i2c.h>
+#include <drivers/gpio.h>
+
+
+
+
+
+static const struct device *gpio_dev;
+static const struct device *i2c_dev;
+
+void main(void)
+{
+	gpio_dev = device_get_binding("FXL6408");
```

Comment:
binding will probably have to be gotten in a different way, most likely something similar to [[this](https://github.com/abelsensors/zephyr/pull/2/files#diff-f41f6a6439299463fc93f46da75c3739400e9935220485ac67013cbb0fe4d2a5R25)]